nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/CloudFrontBinaryManager.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * (C) Copyright 2011-2017 Nuxeo SA (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Arnaud Kervern
+ */
+package org.nuxeo.ecm.core.storage.sql;
+
+import static org.nuxeo.runtime.api.Framework.isBooleanPropertyTrue;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.security.PrivateKey;
+import java.security.spec.InvalidKeySpecException;
+import java.util.Date;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.apache.http.client.utils.URIBuilder;
+import org.nuxeo.ecm.core.blob.ManagedBlob;
+
+import com.amazonaws.AmazonClientException;
+import com.amazonaws.auth.PEM;
+import com.amazonaws.auth.RSA;
+import com.amazonaws.services.cloudfront.CloudFrontUrlSigner;
+import com.amazonaws.services.cloudfront.CloudFrontUrlSigner.Protocol;
+import com.amazonaws.util.IOUtils;
+
+public class CloudFrontBinaryManager extends S3BinaryManager {
+
+    private static final String BASE_PROP = "cloudfront.";
```

Comment:
why ?
I do not want to scope `CloudFront` parameters in the same name as `S3` ones.